external/storm-hdfs/src/main/java/org/apache/storm/hdfs/bolt/AbstractHdfsBolt.java

```diff
@@ -99,6 +109,13 @@ public final void prepare(Map conf, TopologyContext topologyContext, OutputColle
             }
         }
 
+        // If interval is non-zero then it has already been explicitly set and we should not default it
+        if (conf.containsKey("topology.message.timeout.secs") && tickTupleInterval == 0)
+        {
+            Integer topologyTimeout = Integer.parseInt(conf.get("topology.message.timeout.secs").toString());
```

Comment:
Can we please use `Utils.getInt(conf.get(Conf.TOPOLOGY_MESSAGE_TIMEOUT_SECS))` instead.